sphinx_rtd_theme/breadcrumbs.html

```diff
@@ -55,5 +56,14 @@
       </li>
     {% endblock %}
   </ul>
+
+  <div class="rst-breadcrumbs-buttons" role="navigation" aria-label="breadcrumb navigation">
```

Comment:
Hmm. Seems like this should be an option that's enabled with `html_theme_options` -- where you can set it to `[top, bottom, None]`.